Jenkinsfile

```diff
@@ -6,37 +6,52 @@ pipeline{
     buildDiscarder logRotator(artifactDaysToKeepStr: '7', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: '')
   }
   environment{
-    JAVA_HOME_DIR="/home/builder/java/jdk-17.0.2"
+    JAVA17_HOME_DIR="/home/builder/java/jdk-17.0.2"
     GRADLE_DIR   =".gradle"
     DMC_USER     = credentials('MLBUILD_USER')
     DMC_PASSWORD = credentials('MLBUILD_PASSWORD')
+    MARKLOGIC12_IMAGE_TAG = "ml-docker-db-dev-tierpoint.bed-artifactory.bedford.progress.com/marklogic/marklogic-server-ubi-rootless:12.0.0-ubi-rootless-2.2.2"
```

Comment:
I don't think we want this yet because it's suggesting that it's a tag other than what's in .env. Until we have a use case for running the tests against multiple MarkLogic version, leave this out so that the value in .env is used.